src/Playwright/Core/CDPSession.cs

```diff
@@ -50,12 +50,12 @@ public CDPSession(IChannelOwner parent, string guid) : base(parent, guid)
 
     public async Task DetachAsync()
     {
-        await _channel.DetachAsync();
+        await _channel.DetachAsync().ConfigureAwait(false);
     }
 
     public async Task<JsonElement?> SendAsync(string method, Dictionary<string, object>? args = null)
     {
-        return await _channel.SendAsync(method, args);
+        return await _channel.SendAsync(method, args).ConfigureAwait(false);
```

Comment:
Since the method just forwards, it doesn't need to use `async` / `await`:

```c#
    public Task<JsonElement?> SendAsync(string method, Dictionary<string, object>? args = null)
    {
        return _channel.SendAsync(method, args);
    }
```